Userland/Libraries/LibWeb/HTML/BrowsingContext.cpp

```diff
@@ -1292,7 +1292,13 @@ void BrowsingContext::set_system_visibility_state(VisibilityState visibility_sta
     // has changed to newState, it must queue a task on the user interaction task source to update
     // the visibility state of all the Document objects in the top-level browsing context's document family with newState.
     auto document_family = top_level_browsing_context().document_family();
-    queue_global_task(Task::Source::UserInteraction, Bindings::main_thread_vm().current_realm()->global_object(), [visibility_state, document_family = move(document_family)] {
+
+    // From the new navigable version, where it tells us what global object to use here:
+    // 1. Let document be navigable's active document.
+    // 2. Queue a global task on the user interaction task source given document's relevant global object to update the visibility state of document with newState.
+    // FIXME: Update this function to fully match the navigable version.
```

Comment:
`function` = `feature`, right?

Reply:
no, `function` = `function`. `set_system_visibility_state()` to be specific.